### ISSUE TITLE
tables inside tabs 

### ISSUE THREAD

**Issue by xjpmauricio** [markmalek/Fixed-Header-Table#126]
Hi, I have a three tabs (using bootstrap 3) that contain each a table, well formatted etc. Only the first table gets correctly formatted by fixedHeaderTable and, all the others do not get well formatted; the width is wrong, the table header is duplicated, etc. I guess this has something to do with the table not being visible when rendered?
I'm using it in a very simple way:

$('#mytable').fixedHeaderTable({ height: 412, autoShow: true});

**Comment by russgthomas** [comment]
Yep, me too - this is exactly the issue I have: duplicate headers appear, my table is width:100% but doesn't appear 100% of its container, the fHT table *is* 100% but then the fHT column headers don't align with the real table's headers. In dev tools I see that fHT has given my table width:83px - this happens after it tries to figure out the scrollbar's width.  I've given up - too much work.

**Comment by mehrdd** [comment]
+1
I'm frustrated by this bug ...

**Comment by m4ttr33d** [comment]
Did this ever get solved?
